pkg/rtc/utils.go

```diff
@@ -108,15 +108,12 @@ func IsEOF(err error) bool {
 	return err == io.ErrClosedPipe || err == io.EOF
 }
 
-func RecoverSilent() {
-	recover()
-}
-
-func Recover(l logger.Logger) {
+func Recover(l logger.Logger) any {
```

Comment:
what `any` is an accepted type in golang? :TIL:

Reply:
nit: how about returning the error (below) instead. returning `any` isn't as useful.

Reply:
the recover builtin returns any, and I am returning that value.